openeogeotrellis/load_stac.py

```diff
@@ -239,10 +239,18 @@ def operator_value(criterion: Dict[str, object]) -> (str, object):
             else:
                 modifier = None
                 # standard behavior seems to be to include only a minimal subset e.g. https://stac.openeo.vito.be/
-                fields = sorted(
-                    f"properties.{property_name}"
-                    for property_name in {"proj:epsg", "proj:bbox", "proj:shape"}.union(all_properties.keys())
-                )
+                # include all fields to ease the transition of stac catalog
```

Comment:
@bossie , would it make sense to only do this when requesting from https://stac.openeo.vito.be ?